### PR TITLE
feat(cairo): wait for writer if it's about to be released by maintanance from the pool

### PR DESCRIPTION
**Pull request by ideoma**
Few issues with Engine maintanance and Writer pool fixed:

- When `CairoEngine` examines `WriterPool` periodically to evict and close old unused entries it blocks writer for a period of time needed to close the instance. If an `Insert` statement is processed at the same time it will get `writer busy` error with empty reason. To prevent it WriterPool will lock the entries about to be evicted with negative thread id and if such a lock found in `get` method another attempt to re-acquire the writer will be performed.
- When  Writer Pool is examined by `CairoEngine` it will unconditionally unlock all locked table entries. If a table is being created at the time where maintanance the creation fails
- Make busy writer Alter table test more stable in PgWire and Http Json integration tests.

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 27 (85.19%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|26|84.62%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 22 / 27 (81.48%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|21|26|80.77%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 22 / 27 (81.48%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|21|26|80.77%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

com.naver.nid.cover.parser.coverage.exception.ParseException: java.io.FileNotFoundException: /usr/share/vsts-agent/_work/1/s/core/CCReport43F6D5EF/jacoco.xml (No such file or directory)

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

File: core/src/main/java/io/questdb/cairo/CairoEngine.java [hunk 1]
```diff
@@ -525,7 +523,7 @@ protected boolean runSerially() {
                 // process and drain cmd queue
                 useful = true;
             }
-            if (last + checkInterval < t) {
+            if (last + configuration.getIdleCheckInterval() < t) {
```
Comment (ideoma) [inline core/src/main/java/io/questdb/cairo/CairoEngine.java:526]: Better to not copy the value for lifetime but get it from config for testing and if we will want to ever make configuration dynamic

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

File: core/src/main/java/io/questdb/cairo/CairoEngine.java (hunk 1 above)
Comment (bluestreak01) [reply core/src/main/java/io/questdb/cairo/CairoEngine.java:526]: this is premature, when we come to define dynamic configuration we will be revisiting values one by one. This change doesn't solve the issue of dynamic configuration but potentially adds a gotcha should latency of `getIdleCheckInterval()` increase

Comment (ideoma) [reply core/src/main/java/io/questdb/cairo/CairoEngine.java:526]: I think we can accept this "potential" latency given it's not in a tight loop and not on hot path for sake of mocking value in the tests, cannot we?

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

File: core/src/main/java/io/questdb/cairo/CairoEngine.java (hunk 1 above)
Comment (ideoma) [reply core/src/main/java/io/questdb/cairo/CairoEngine.java]: ok, I reverted this change

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
Should be good to go. I'll trigger another build on ci just in case

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 23 / 28 (82.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 28 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 37 / 41 (90.24%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 36 / 41 (87.80%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 43 / 47 (91.49%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 42 / 47 (89.36%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 50 / 54 (92.59%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 50 / 54 (92.59%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 49 / 54 (90.74%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 50 / 54 (92.59%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 49 / 54 (90.74%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 50 / 54 (92.59%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 49 / 54 (90.74%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 50 / 54 (92.59%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|23|27|85.19%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 49 / 54 (90.74%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/pool/WriterPool.java|22|27|81.48%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGWireServer.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/QueryFutureUpdateListener.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogAlertSocket.java|13|13|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

com.naver.nid.cover.parser.coverage.exception.ParseException: java.io.FileNotFoundException: /home/vsts/work/1/s/core/CCReport43F6D5EF/jacoco.xml (No such file or directory)

**Comment by ideoma** [comment]
#### [PR Coverage check]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

com.naver.nid.cover.parser.coverage.exception.ParseException: java.io.FileNotFoundException: /home/vsts/work/1/s/core/CCReport43F6D5EF/jacoco.xml (No such file or directory)

**Comment by ideoma** [comment]
#### [PR Coverage check]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

com.naver.nid.cover.parser.coverage.exception.ParseException: java.io.FileNotFoundException: /usr/share/vsts-agent/_work/1/s/core/CCReport43F6D5EF/jacoco.xml (No such file or directory)

**Comment by ideoma** [comment]
#### [PR Coverage check]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

com.naver.nid.cover.parser.coverage.exception.ParseException: java.io.FileNotFoundException: /home/vsts/work/1/s/core/CCReport43F6D5EF/jacoco.xml (No such file or directory)

**Comment by ideoma** [comment]
#### [PR Coverage check]

coverage check fail. please retry. :fearful:

[Please let me know](https://github.com/naver/cover-checker/issues/new) when error again.

com.naver.nid.cover.parser.coverage.exception.ParseException: java.io.FileNotFoundException: /home/vsts/work/1/s/core/CCReport43F6D5EF/jacoco.xml (No such file or directory)

**Pull request closed without merging**